apps/files/src/components/AllFilesList.vue

```diff
@@ -38,7 +38,7 @@
           class="uk-margin-small-left"
         />
       </div>
-      <div class="uk-flex uk-flex-middle">
+      <div v-if="!$_isFavoritesList" class="uk-flex uk-flex-middle">
```

Comment:
Why hiding the indicators in the favourites list?

Reply:
we forgot to hide them before like in the other lists

reason is that we cannot show indirect share indicators for the whole list as it would require loading the sharesTree for every item, very very expensive

Reply:
Ok, thanks!